Add additional docs for substance designer (#338)
ynput/ayon-documentation: website/docs/addon_substancedesigner_artist.md

## Publishing Textures

> Users need to open the graph(s) of which inherited from the package they want to publish,

Reviewer: ```suggestion
Users must open the graph(s) inherited from the package they intend to publish,
```

and go to **AYON -> Create** to create texture instance.

> Users can define which graphs and which graph outputs they want to export.

Reviewer: ```suggestion
Users can define which graphs and its output(s) they want to export.
```

from the default pattern set in export presets.
So `$(graph)_$(identifier)` becomes `basecolor`.

## Publishing Sbsar

> Users need to open the graph(s) of which are inherited from the package

Reviewer: ```suggestion
Users must open the graph(s) inherited from the package
```
